packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -113,6 +120,7 @@ const getStyles = (
 ) => {
   const color = theme.colors[severity];
   const borderRadius = theme.shape.borderRadius();
+  const borderColor = tinycolor2(color.border).setAlpha(0.2).toString();
```

Comment:
This just made me realize that we actually use `setAlpha` in a bunch of places, so we need to remember this during the color tokens exploration so we can take these into account.

Reply:
@JoaoSilvaGrafana yes, if we need this weak colored border in many places it should be added as a token in theme. but so far this and the Badge is are the only two I know of we do this with the border color .